tests/common/jormungandr/logger.rs

```diff
@@ -0,0 +1,34 @@
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use std::path::PathBuf;
+
+#[derive(Debug)]
+pub struct JormungandrLogger {
+    pub log_file_path: PathBuf,
+}
+
+impl JormungandrLogger {
+    pub fn new(log_file_path: PathBuf) -> Self {
+        JormungandrLogger { log_file_path }
+    }
+
+    pub fn get_lines_with_error(&self) -> Vec<String> {
```

Comment:
Similarly, this could return `impl Iter<String>` and let the caller decide whether to buffer the data or process it iteratively.